### PR TITLE
Verify docker image latest tag - Fix

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
Ready

## Description
Script & Integration files have dockerimage in different places.

## Does it break backward compatibility?
   - No

## Must have
- [ ] Code Review

### CODE REVIEW

File: Tests/scripts/hook_validations/docker.py [hunk 1]
```diff
@@ -225,8 +232,8 @@ def parse_docker_image(docker_image):
         """
         if docker_image:
             try:
-                tag = re.findall(r'(demisto\/.+):.+', docker_image, re.IGNORECASE)[0]
-                name = re.findall(r'demisto\/.+:(.+)', docker_image, re.IGNORECASE)[0]
+                name = re.findall(r'(demisto\/.+):.+', docker_image, re.IGNORECASE)[0]
+                tag = re.findall(r'demisto\/.+:(.+)', docker_image, re.IGNORECASE)[0]
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/docker.py]: why is that separated to two why not `name, tag = re.findall(r'(demisto\/.+):(.+)', docker_image, re.IGNORECASE)[0]`

**Comment by Itay4** [comment]
why dont we have a test for this?

File: Tests/scripts/hook_validations/docker.py (hunk 1 above)
Comment (guyfreund) [reply Tests/scripts/hook_validations/docker.py]: fixed

**Comment by guyfreund** [comment]
> why dont we have a test for this?

now there is

File: Tests/scripts/hook_validations/docker.py [hunk 2]
```diff
@@ -225,9 +236,8 @@ def parse_docker_image(docker_image):
         """
         if docker_image:
             try:
-                name = re.findall(r'(demisto\/.+):.+', docker_image, re.IGNORECASE)[0]
-                tag = re.findall(r'demisto\/.+:(.+)', docker_image, re.IGNORECASE)[0]
-                return name, tag
+                regex = re.findall(r'(demisto\/.+):(.+)', docker_image, re.IGNORECASE)
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/docker.py]: `re.findall` returns list of tuples
```suggestion
                regex = re.findall(r'(demisto\/.+):(.+)', docker_image, re.IGNORECASE)[0]
```

Comment (guyfreund) [reply Tests/scripts/hook_validations/docker.py]: fixed even though test doesn't fails :D

**Pull request merged**